pkg/models/model.go

```diff
@@ -123,14 +123,14 @@ type VersionResponse struct {
 type ModelResponse struct {
 
 	// Model unique ID
-	Id int32
+	Id uint64
 
 	// Model name
 	Name string
 
 	FullName string
 
-	CVTask string
+	Task string
 
 	Versions []VersionResponse
 }
```

Comment:
The JSON response is not corresponding to the OpenAPI format:
```
{
  "model": {
    "id": "string",
    "name": "string",
    "fullName": "string",
    "task": "TASK_UNSPECIFIED",
    "modelVersions": [
      {
        "version": "string",
        "modelId": "string",
        "description": "string",
        "createdAt": "2022-03-20T19:38:16.387Z",
        "updatedAt": "2022-03-20T19:38:16.387Z",
        "status": "STATUS_UNSPECIFIED"
      }
    ]
  }
}
```

Reply:
@pinglin I got your point. Yes, should be use the same struct